src/Microsoft.TestPlatform.PlatformAbstractions/net451/System/ProcessHelper.cs

```diff
@@ -15,21 +17,98 @@ namespace Microsoft.VisualStudio.TestPlatform.PlatformAbstractions;
 
 public partial class ProcessHelper : IProcessHelper
 {
+    [DllImport("kernel32.dll", SetLastError = true)]
+    private static extern bool IsWow64Process2(IntPtr process, out ushort processMachine, out ushort nativeMachine);
```

Comment:
Looks like the rule is not active but might still be worth to follow it [CA1060](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1060). WDYT?

Reply:
done